spec/controllers/election/candidates_controller_spec.rb

```diff
@@ -0,0 +1,156 @@
+require 'rails_helper'
+
+RSpec.describe Election::CandidatesController, type: :controller do
+
+  # Override get_commit
+  before(:each) do
+    allow_any_instance_of(ApplicationController).to receive(:get_commit).and_return(true)
+  end
+  # Build an election
+  let(:election) { create(:election) }
+  before(:each) do
+    allow(Election).to receive(:current).and_return(election)
+  end
+  # Testing objects
+  let(:user) { create(:user) }
+  let(:search_post) { create(:post) }
+  let(:not_owner) { create(:user) }
+  let(:candidate) { create(:candidate, profile: user.profile,
+                           election: election, post: search_post) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 19, 67 should be on its own line.